### PR TITLE
Philip Medlin

### PR DESCRIPTION
**Pull request by PhiMed**
Here is my prework!

My Gear Up gist is at:
https://gist.github.com/PhiMed/47533c7ca46a229da6470227a7663835

### CODE REVIEW

File: final_prep/annotations.rb [hunk 1]
```diff
@@ -3,41 +3,70 @@
...
+#Execute the build_a_bear method using the following respective arguments: 'Fluffy', etc
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+#Execute the same method again with these new arguments
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
+#Note that the method returns the built_bear value but doesn't print anything to the terminal
 
 
 # FizzBuzz
-
+#Define a new method called fizzbuzz that takes 3 arguments (num1, num2, range)
```
Comment (mschae16) [inline final_prep/annotations.rb:42]: One quick note - `fizzbuzz` and `build_a_bear` are actually functions, not methods. Methods are functions that belong to a class.

File: final_prep/annotations.rb [hunk 2]
```diff
@@ -3,41 +3,70 @@
...
+#...print the value of i to the terminal
       puts i
+#end the conditional
     end
+#end the iteration
   end
+#end the method
 end
-
+#Execute the method fizzbuzz with the arguments 3, 5, and 100 (100 being the range to go up to and the others being the divisors in the if/elsif processes)
```
Comment (mschae16) [inline final_prep/annotations.rb:69]: Really thorough annotations!

File: final_prep/mod_zero_hero.rb [hunk 3]
```diff
@@ -1,49 +1,96 @@
...
 
+arch_enemies.push("Jean Charles")
+
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
 
+puts arch_enemies.inspect
+
 # Remove the first sidekick from the sidekicks array
 
+sidekicks.delete_at(0)
```
Comment (mschae16) [inline final_prep/mod_zero_hero.rb:57]: Another option here is `sidekicks.shift`

**Review by mschae16 — APPROVED** [review]
Excellent job on this prework, @PhiMed  - you are officially `Turing ready`! Let me know if you have any questions, and keep up the great work.

**Comment by PhiMed** [comment]
error

**Pull request closed without merging**